templates/dashboard/alias_contact_manager.html

```diff
@@ -64,7 +64,11 @@ <h1 class="h3"> {{ alias.email }} contacts
         <div class="small-text">
           Where do you want to send the email?
         </div>
-        <button class="btn btn-primary mt-2">Create reverse-alias</button>
+          {% if alias.user.is_premium() %}
```

Comment:
Should we use the global flag here so free users can still create reverse alias in the meantime?

Reply:
Good catch. I've added a var in the template with the same condition we use in the logic.